pandas/_libs/tslibs/timedeltas.pyx

```diff
@@ -1127,8 +1127,7 @@ class Timedelta(_Timedelta):
         'ms', 'milliseconds', 'millisecond', 'milli', 'millis', 'L',
         'us', 'microseconds', 'microsecond', 'micro', 'micros', 'U',
         'ns', 'nanoseconds', 'nano', 'nanos', 'nanosecond', 'N'}
-    days, seconds, microseconds,
-    milliseconds, minutes, hours, weeks : numeric, optional
+    days,seconds,microseconds,milliseconds,minutes,hours,weeks : numeric, optional:
```

Comment:
Does this pass LINTing? May or may not be an issue just not aware of combining parameters like this into one line anywhere else

Reply:
Hmm OK. Technically the signature for this class only accepts **kwargs, so I would use that as the type and then move everything about days, seconds, etc... down into the descption